tests/test-pool-search.rb

```diff
@@ -17,44 +17,35 @@
   url_list = [ catalog_url, jmrl_url, article_url ]
 
   before do
-    #Airborne.configuration.base_url = article_url
+    Airborne.configuration.base_url = ENV['URL']
     Airborne.configuration.headers = { 'Content-Type' => 'application/json', 'Accept' => 'application/json', 'Authorization' => 'Bearer bkb4notbo1bc80d2uucg' }
     Airborne.configuration.verify_ssl = false
   end
-
   #
   # tests that the search call returns reported JSON
   #
-  it 'search should return json' do
-    url_list.each do |url|
-      Airborne.configuration.base_url = url
+  it "#{ENV['URL']} search should return json" do
```

Comment:
Could we have a variable initialized to the value ENV['URL'] to avoid having it all over the place.  It will save time if we need/want to change the environment variable name.